jenkins/image_building/dib_elements/ubuntu-ci/post-install.d/01_install

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+sudo apt-get update
+sudo apt-get install -y \
+  coreutils \
+  apt-transport-https \
+  ca-certificates \
+  gnupg-agent \
+  software-properties-common \
+  openssl \
+  python-is-python3 \
+  chrony \
+  qemu
+
+# Configure
+sudo chronyc -a 'burst 4/4' && sudo chronyc -a makestep
+sudo systemctl enable chrony
+sudo systemctl start chrony
+
+# Enable nested virtualization
+sudo bash -c 'cat << EOF > /etc/modprobe.d/qemu-system-x86.conf
+options kvm-intel nested=y enable_apicv=n
+EOF'
+echo "Reboot required"
+
+# Install Docker
```

Comment:
Why do we install docker here, if it was installed in the base element?

Reply:
As I mention above, this installed in a parent OS.

Reply:
Not sure I followed? What is "a parent OS"?

Reply:
it is the vm we are running, not the one we are building.

Reply:
Hmm. Shouldn't `docker` have already been there in the VM image?
Also, I did not know that a script could install anything on the host machine, while it's a script inside a DIB element? Could you please elaborate on this topic a little?